src/exact/ExactMapper.cpp

```diff
@@ -5,6 +5,12 @@
 
 #include "exact/ExactMapper.hpp"
 
+#include "Encodings/Encodings.hpp"
+#include "LogicBlock/LogicBlock.hpp"
+#ifdef Z3_FOUND
```

Comment:
As stated earlier, this compile definition should be set by the LogicBlocks module whenever Z3 is available.